src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -550,51 +588,10 @@ bool isSupportedConversion(Conversion conversion, BoundExpression operandOpt)
             }
         }
 
-        private new void VisitLvalue(BoundExpression node)
```

Comment:
>VisitLvalue [](start = 25, length = 11)

:memo: The role of VisitLValue is now handled by regular Visit, by storing a l-value type in the result that we bubble up.